### ISSUE TITLE
HECNN on TinyImageNet: EL performs worse than EE

### ISSUE THREAD

**Issue by GitZH-Chen** [kschwethelm/HyperbolicCV#4]
Hi,

Thanks for the great work.

I trained the HECNN model on TinyImageNet for image classification using the same configurations as in EL-ResNet18.txt and E-ResNet18.txt.
However, the validation accuracy is already much lower in the early epochs, as shown in the attached figure.

I also noticed that another user has reported a similar issue.
Could you please check whether the TinyImageNet setup or any hyperbolic settings have changed compared to the paper?

Thanks.

<img width="1082" height="386" alt="Image" src="https://github.com/user-attachments/assets/dd87160f-a184-4c00-8291-26218f5c1272" />

PS: I can reproduce the CIFAR-10 and CIFAR-100 HECNN classification experiments.

**Comment by GitZH-Chen** [comment]
Hi, this issue can be closed. Using your training scripts and configs, the Hybrid Lorentz setup (ResNet-18 + LMLR) will finally reach results similar to the paper.
